### PR TITLE
[Docs] Improve REST API Docs Legibility

### LINKED ISSUE #7365: [Docs] Improve REST API Docs Legibility
#### Current State
A bit of CSS here would go a long way on legibility and usefulness of the [REST API docs](https://docs.meshery.io/reference/rest-apis).

#### Desired State

1. - [ ] [@senali-d] Improve [mockup of these API docs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=2191%3A12342) to use brand-appropriate colors.
1. - [ ] [@senali-d] Improve information architecture in the Description (too much bold text, not enough bullets, no `<code>` markup, no hyperlinks, and so on).
1. - [ ] Add a table of contents.
1. - [ ] Sort endpoints alphabetically.
1. - [ ] Grouping endpoints by path hierarchy; e.g. all /api/meshmodel endpoints adjacent to one another

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by senali-d**
**Notes for Reviewers**

This PR fixes #7365

Completed bellow desired state in this PR

1. Improve [mockup of these API docs](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=2191%3A12342) to use brand-appropriate colors.
2. Improve information architecture in the Description (too much bold text, not enough bullets, no code ```<code>``` markup, no hyperlinks, and so on).

Current state.

image

image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5ae9260648d085df2452634697a14bc39ddd3b7d at: https://6416ced10327f1181607b76e--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@ritiksaxena124 would you like to review?

**Comment by leecalcote** [comment]
Thanks, @senali-d!

The greens used here are not within brand coloring.

**Comment by l5io** [comment]
🚀 Preview for commit bf663aa2436a06ace11acbff59edd586d05dbd2d at: https://641861449d0f4346e68a698e--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
Current: 

Screenshot 2023-03-20 191746

Your version:

Screenshot 2023-03-20 191730

**Comment by senali-d** [comment]
@vishalvivekm Thanks for the feedback.
OK. Let's keep them as it is in the current version.

**Comment by l5io** [comment]
🚀 Preview for commit 8b2e770fa8ab24e2f8346df71d8b5c2308fbbb82 at: https://6418854dc816976a9b3a490d--meshery-docs-preview.netlify.app

File: docs/_data/swagger.yml [hunk 1]
```diff
@@ -1553,8 +1610,7 @@ responses:
             subType:
                 type: string
         description: |-
-            https://docs.google.com/drawings/d/1_qzQ_YxvCWPYrOBcdqGMlMwfbsZx96SBuIkbn8TfKhU/edit?pli=1
-            see RELATIONSHIPDEFINITIONS table in the diagram
+            [see RELATIONSHIPDEFINITIONS table in the diagram](https://docs.google.com/drawings/d/1_qzQ_YxvCWPYrOBcdqGMlMwfbsZx96SBuIkbn8TfKhU/edit?pli=1)
```
Comment (leecalcote) [inline docs/_data/swagger.yml:1613]:
```suggestion
`Relationships` define the genealogy between one or more `Components` interconnected `Components`. Just as in familial relationships, MeshModel `Relationships` are represented in a variety of forms very much resembling familiar ancestral lineage, including `hierarchical` relationships such as direct parent and child relationship as well as any number of layers of indirect grandparent and grandchild relationships. Peer relationships like that of `sibling`s take on different types such as `network` and `dependency`.  Each type of `Relationship` has `metadata`, `selectors` and some optional parameters. 
```

**Review by leecalcote — DISMISSED** [review]

**Comment by leecalcote** [comment]
Thanks, @senali-d 👍

**Comment by ritiksaxena124** [comment]
hey, @senali-d you did great work, but we still have some typo issues. it would be great if you fix those as well

issue

type-issueJPG

**Comment by senali-d** [comment]
Sure. I'll fix them.

**Comment by l5io** [comment]
🚀 Preview for commit f400c4b694adfa8093b0194ea049abc3441b0c84 at: https://641afd2d2268470830e66d3a--meshery-docs-preview.netlify.app

**Comment by senali-d** [comment]
@ritiksaxena124 Fixed typo issues mention in the above comment.

**Comment by leecalcote** [comment]
Merge conflicts...

**Comment by goldfishdolphin** [comment]
Great work @senali-d 👏

**Comment by l5io** [comment]
🚀 Preview for commit 09cb170c1475a2b4fa5cbde30b301a509111f16a at: https://6421cc42e3015f164f822211--meshery-docs-preview.netlify.app

**Comment by senali-d** [comment]
@vishalvivekm Any feedback?

**Comment by vishalvivekm** [comment]
> @vishalvivekm Any feedback?

Yeah, I'll be back with 'em soon, a lot to look in so.. :)

**Comment by leecalcote** [comment]
Eliminating merge conflict.

**Comment by l5io** [comment]
🚀 Preview for commit 17ab7224ee8a4a164a328e5abb749e8c67dcbb31 at: https://64261e7c1f98421e03eec720--meshery-docs-preview.netlify.app

**Pull request merged**